lib/fs.js

```diff
@@ -196,10 +196,14 @@ function Stats(
   this.ino = ino;
   this.size = size;
   this.blocks = blocks;
-  this._atim_msec = atim_msec;
-  this._mtim_msec = mtim_msec;
-  this._ctim_msec = ctim_msec;
-  this._birthtim_msec = birthtim_msec;
+  this.atim_msec = atim_msec;
```

Comment:
I guess, yeah. If you want to go with `mtimeMs`, I’m not going to stand in your way or anything :)